### PR TITLE
Woovi backend

### PR DESCRIPTION
**Pull request by grandehe4rt**

### CODE REVIEW

File: src/database/connection.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const URI = process.env.MONGODB_URI;
+
+export const databaseConnection = () => {
+    if (URI != undefined) {
```
Comment (fersilva16) [inline src/database/connection.ts:9]: prefer `!==`

Comment (fersilva16) [reply src/database/connection.ts:9]: Prefer early return

File: src/database/connection.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        mongoose.connect(URI);
```
Comment (fersilva16) [inline src/database/connection.ts:10]: missing await

**Review by fersilva16 — APPROVED** [review]

**Review by grandehe4rt — COMMENTED** [review]
Making database connection more readable

**Review by daniloab — APPROVED** [review]

**Pull request merged**